src/wifi/AdafruitIO_AIRLIFT.h

```diff
@@ -158,6 +158,9 @@ class AdafruitIO_AIRLIFT : public AdafruitIO {
     /**************************************************************************/
     void _connect()
     {
+      if(strlen(_ssid) != 0)
+      {
+        _disconnect();
       // setup ESP32 pins
       if (_ssPin != -1) {
           WiFi.setPins(_ssPin, _ackPin, _rstPin, _gpio0Pin, _wifi);
```

Comment:
Why is `WiFi.setPins` occurring here? Do you mean to add a `}` after the call to `_disconnect()`?

Reply:
The  } is down at the end of the function. The wiFi.setPins, etc. is part of the original code, left unindented so the diff shows no changes.  I'll fix the indent now so it will be clearer to read.

Reply:
I think that code for the airlift may be in error with the original -- trying a compile of example 00 with the current release code and #define USE_AIRLIFT uncommented produces compile errors under both ESP32 and M0.  I'm not familiar with the airlift devices and was just mimicking what worked with the M0 WINC1500 setup.

Reply:
What compile errors are you seeing? Travis seems to be passing with commit a1ac2a2.

Reply:
```

In file included from /Users/sellensr/Documents/Arduino/libraries/Adafruit_IO_Arduino/src/AdafruitIO_WiFi.h:22:0,
                 from sketch/config.h:32,
                 from /var/folders/3w/fph9nxcx59bgndf_ydyvphlr0000gn/T/arduino_modified_sketch_795096/adafruitio_00_publish.ino:18:
/Users/sellensr/Documents/Arduino/libraries/Adafruit_IO_Arduino/src/wifi/AdafruitIO_AIRLIFT.h: In member function 'virtual void AdafruitIO_AIRLIFT::_connect()':
/Users/sellensr/Documents/Arduino/libraries/Adafruit_IO_Arduino/src/wifi/AdafruitIO_AIRLIFT.h:166:18: error: 'class WiFiClass' has no member named 'setPins'
             WiFi.setPins(_ssPin, _ackPin, _rstPin, _gpio0Pin, _wifi);
                  ^~~~~~~
Multiple libraries were found for "SPI.h"
 Used: /Users/sellensr/Library/Arduino15/packages/adafruit/hardware/samd/1.5.7/libraries/SPI
Multiple libraries were found for "Adafruit_ZeroDMA.h"
 Used: /Users/sellensr/Library/Arduino15/packages/adafruit/hardware/samd/1.5.7/libraries/Adafruit_ZeroDMA
Multiple libraries were found for "AdafruitIO_WiFi.h"
 Used: /Users/sellensr/Documents/Arduino/libraries/Adafruit_IO_Arduino
Multiple libraries were found for "Adafruit_MQTT.h"
 Used: /Users/sellensr/Documents/Arduino/libraries/Adafruit_MQTT_Library
Multiple libraries were found for "ArduinoHttpClient.h"
 Used: /Users/sellensr/Documents/Arduino/libraries/ArduinoHttpClient
Multiple libraries were found for "WiFiNINA.h"
 Used: /Users/sellensr/Documents/Arduino/libraries/WiFiNINA
exit status 1
Error compiling for board Adafruit Feather M0.
```